ros2_control_test_nodes/ros2_control_test_nodes/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2021 Stogl Robotics Consulting UG (haftungsbeschränkt)
```

Comment:
do we really need to litter in the init.py files too??

Reply:
Unfortunately, yes....